src/styles/_media.scss

```diff
@@ -0,0 +1,20 @@
+@media (max-width: 1023px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
would be good to move width to a variable

Reply:
not implemented